pages/flashbar/interactive.page.tsx

```diff
@@ -13,7 +13,8 @@ export default function InteractiveFlashbar() {
   };
 
   const add = (type: FlashbarProps.Type, hasHeader = false) => {
-    setItems(items => [generateItem({ type, id: nextId.current.toString(), dismiss, hasHeader }), ...items]);
```

Comment:
This makes sure that the new item has the correct id and not the next one. Since React batches state updates, the old logic could lead to two items having the same id.